packages/ember-css-modules/lib/modules-preprocessor.js

```diff
@@ -223,8 +223,7 @@ function formatJS(classMapping) {
 
 class ModuleSourceFunnel extends Funnel {
   constructor(input, stylesTree, options) {
-    super(input, options);
-    this.stylesTree = stylesTree;
+    super([input, stylesTree], options);
```

Comment:
Just based on my own understanding, I would have expected the change you made to `Funnel` to work exactly how you described, but it looks like this _does_ end up having an effect on the base class's behavior.

I haven't dived super deep, but I'm seeing contents from both trees show up in `this.input.entries(...)`, which seems to cause [the funnel to try and `lstat` an ultimately nonexistent file](https://github.com/salsify/ember-css-modules/pull/239/checks?check_run_id=2847189507#step:6:90) from the second input tree.

Reply:
Ya, this is definitely true. I replicated that specific error and fixed it over in https://github.com/broccolijs/broccoli-funnel/pull/147.

Reply:
I've updated this PR with that change, which should avoid the error we see in CI.